apiserver/admin.go

```diff
@@ -65,6 +65,12 @@ func (a *admin) login(req params.LoginRequest, loginVersion int) (params.LoginRe
 
 	if a.root.state.IsCAAS() {
 		// XXX don't bother with actual login checks here. Just fake it for the prototype for now.
+		u, err := a.srv.state.User(names.NewUserTag("admin"))
```

Comment:
Looking at this further, adding a FindEntity is easy, however that then needs a st.User (or equivalent), which then means that we need type User to work without being tied to a *state.State... I'd say land this and I'll come back to User soon.